src/foam/nanos/notification/email/DAOResourceLoader.java

```diff
@@ -43,7 +43,7 @@ public static EmailTemplate findTemplate(X x, String name, String groupId) {
       }
 
       Group group = (Group) groupDAO.find(groupId);
-      groupId = ( group != null && ! SafetyUtil.isEmpty(group.getParent()) ) ? group.getParent() : "*";
+      groupId = ( group != null && ! SafetyUtil.isEmpty(group.getParent().toString()) ) ? group.getParent().toString() : "*";
```

Comment:
Why is this change needed?

Reply:
getParent() already returns a String so the toString() isn't needed.